### PR TITLE
Disable DuckDB result check for Spark max_by and min_by functions

### PR DESCRIPTION
**Pull request by rui-mo**
Fixes https://github.com/facebookincubator/velox/issues/7677.

### CODE REVIEW

File: velox/exec/tests/SparkAggregationFuzzerTest.cpp [hunk 1]
```diff
@@ -71,6 +71,10 @@ int main(int argc, char** argv) {
   size_t initialSeed = FLAGS_seed == 0 ? std::time(nullptr) : FLAGS_seed;
   auto duckQueryRunner =
       std::make_unique<facebook::velox::exec::test::DuckQueryRunner>();
+  duckQueryRunner->disableAggregateFunctions({
+      "max_by", // DuckDB results on equality handling is different with Spark.
```
Comment (mbasmanova) [inline velox/exec/tests/SparkAggregationFuzzerTest.cpp:75]: > // DuckDB results on equality handling is different with Spark.

I don't understand this comment and wonder if other readers will be able to understand it. Perhaps, replace it with a link to GitHub issue that provides more context.

```
  // https://github.com/facebookincubator/velox/issues/7677
  "max_by",
  "min_by",
```

**Comment by mbasmanova** [comment]
@rui-mo Rui, wondering if we are making some progress on replacing DuckDB with Spark in the fuzzers.

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Comment by rui-mo** [comment]
> @rui-mo Rui, wondering if we are making some progress on replacing DuckDB with Spark in the fuzzers.

@mbasmanova Masha, thank you for reminding. @zhztheplayer will take a look with me to find a feasible way, including how to submit job to Spark, and how to fetch result from Spark. cc @zhouyuan

File: velox/exec/tests/SparkAggregationFuzzerTest.cpp (hunk 1 above)
Comment (rui-mo) [reply velox/exec/tests/SparkAggregationFuzzerTest.cpp]: Fixed, thanks.

**Pull request closed without merging**